megamek/src/megamek/client/ui/swing/GUIPreferences.java

```diff
@@ -249,7 +256,39 @@ public class GUIPreferences extends PreferenceStoreProxy {
     public static String RAT_YEAR_MAX = "RATYearMax";
     public static String RAT_PAD_BV = "RATPadBV";
     public static String RAT_SELECTED_RAT = "RATSelectedRAT";
-    
+
+    private static final Color DEFAULT_WHITE = new Color(255, 255, 255);
+    private static final Color DEFAULT_BLACK = new Color(0, 0, 0);
```

Comment:
I don't understand the splits here, as they are just Color.WHITE and Colour.BLACK.